shared/profile/prove-enter-username-container.js

```diff
@@ -35,7 +35,7 @@ export default connector.connect(
     }
 
     return {
-      canContinue: profile.usernameValid,
+      canContinue: true,
```

Comment:
yeah i think cause this was busted in redux. the value of that is supposed to be updated as you type so if you actually enter a valid bitcoin address it'll turn enabled on (and the button as well)